docs/src/test/scala/docs/http/scaladsl/server/directives/JsonStreamingExamplesSpec.scala

```diff
@@ -235,7 +239,7 @@ class JsonStreamingExamplesSpec extends RoutingSpec with CompileOnlySpec {
 
     Post("/metrics", entity = data) ~> route ~> check {
       status should ===(StatusCodes.OK)
-      responseAs[String] should ===("""{"msg":"Total metrics received: 2"}""")
+      responseAs[String] should ===("Total metrics received: 2")
```

Comment:
Another suspicious change

Reply:
this does seem like an important change - I think we need to understand why this needs to change

Reply:
Agreed, it basically does see like some test/business logic was updated and not related to scala 3 at all.

Reply:
Pinging @raboof

Reply:
(counterpart of https://github.com/apache/incubator-pekko-http/pull/150#discussion_r1206637535)